utils/environment.sh

```diff
@@ -28,7 +28,7 @@ fi
 
 # misc
 export URAY_PART_YAML="${URAY_DATABASE_DIR}/${URAY_DATABASE}/${URAY_PART}/part.yaml"
-export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:$PYTHONPATH"
+export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:${URAY_DIR}/third_party/prjuray-tools:$PYTHONPATH"
```

Comment:
This entire line is likely no longer required, as the venv should already have fasm and prjuray-tools? I think this is leftover techdebt from before the venv was created?

Reply:
Same here. Fixed.